### PR TITLE
moving dcomp++ outside MFIter loop

### PR DESCRIPTION
**Pull request by RevathiJambunathan**
This PR fixes a bug in FieldIO.cpp that will address Issue #649

### CODE REVIEW

File: Source/Diagnostics/FieldIO.cpp [hunk 1]
```diff
@@ -636,9 +636,9 @@ WarpX::AverageAndPackFields ( Vector<std::string>& varnames,
 #pragma omp parallel
 #endif
                 for (MFIter mfi(mf_avg[lev]); mfi.isValid(); ++mfi) {
-                    (mf_avg[lev])[mfi].setVal(static_cast<Real>(ParallelDescriptor::MyProc()),
-                                              dcomp++);
+                    (mf_avg[lev])[mfi].setVal(static_cast<Real>(ParallelDescriptor::MyProc()));
                 }
+                dcomp++;
```
Comment (RevathiJambunathan) [inline Source/Diagnostics/FieldIO.cpp]: @WeiqunZhang @atmyers  Can I move
 (mf_avg[lev])[mfi].setVal(static_cast<Real>(ParallelDescriptor::MyProc())) 
outside the MFIter loop and instead do the following:
mf_avg[lev].setVal(static_cast<Real>(ParallelDescriptor::MyProc()));

**Comment by RevathiJambunathan** [comment]
Travis-CI terminated with the following error : 
The job exceeded the maximum time limit for jobs, and has been terminated.

**Comment by MaxThevenet** [comment]
Just re-built the TravisCI tests, now they pass.

**Comment by dpgrote** [comment]
Oops, sorry about that. Good catch finding it.

**Pull request merged**